src/app/config.rs

```diff
@@ -24,6 +24,15 @@ impl AppChainConfig {
     pub fn to_toml(&self) -> Result<String, Error> {
         toml::to_string(self)
     }
+    pub fn fund_msg(&self) {
```

Comment:
i don't think this should be a function of the `AppChainConfig`. let's move this to init.rs itself and accept the DaLayer as a param?